src/data/javascript-quiz.ts

```diff
@@ -1136,7 +1213,19 @@ const javascriptQuiz = [
     Distractor3: "Boolean",
     Explanation:
       "In JavaScript, typeof null is an object.This result of typeof null is actually a bug in the language.",
-    Link: "https://www.freecodecamp.org/news/javascript-typeof-how-to-check-the-type-of-a-variable-or-object-in-js/",
+    Link:
+      "https://www.freecodecamp.org/news/javascript-typeof-how-to-check-the-type-of-a-variable-or-object-in-js/",
+  },
+  {
+    Question: "let a; In JavaScript, what will typeof a return?",
```

Comment:
Code examples are tricky because they have to be all on one line and the wording has to flow nicely. 
Let's go with this rewording so it flows better. 

```suggestion
    Question: "In JavaScript, what will the following code return: typeof a?",
```